build.gradle

```diff
@@ -16,85 +16,62 @@
 
 plugins {
 	id 'java'
-    id 'java-library'
+	id 'java-library'
 	id 'application'
 	id 'com.palantir.docker' version '0.34.0'
 }
 
-java {
-	sourceCompatibility = JavaVersion.VERSION_11
-	targetCompatibility = JavaVersion.VERSION_11
-}
-
 group 'com.exactpro.th2'
 version = release_version
 
-ext {
-	grpcVersion = '1.52.1'
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
 
+ext {
 	mainRsrcDir = 'src/main/resources'
 	sharedDir = file("${projectDir}/shared")
 }
 
 repositories {
-	
-	maven {
-		name 'MavenLocal'
-		url sharedDir
-	}
-
+	mavenCentral()
 	maven {
 		name 'Sonatype_snapshots'
 		url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
 	}
-
 	maven {
 		name 'Sonatype_releases'
 		url 'https://s01.oss.sonatype.org/content/repositories/releases/'
 	}
 
-	mavenCentral()
-
 	configurations.configureEach {
 		resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
 		resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
 	}
 }
 
 dependencies {
-	api platform('com.exactpro.th2:bom:4.0.2')
-
-	compile "org.slf4j:slf4j-api"
-	compile "org.slf4j:slf4j-log4j12"
-
-	compile "io.grpc:grpc-protobuf"
-	compile "io.grpc:grpc-stub"
-	compile "io.grpc:grpc-core"
-	compile "io.grpc:grpc-netty"
+	api platform('com.exactpro.th2:bom:4.1.0')
 
-	compile "com.rabbitmq:amqp-client"
+	implementation "org.apache.logging.log4j:log4j-slf4j2-impl"
+	implementation 'org.apache.logging.log4j:log4j-core'
```

Comment:
I don't think we need those dependencies here. The logging implementation is provided by common library. If it is not, check the version of common library. Maybe you need to update it